app/controllers/event.js

```diff
@@ -0,0 +1,121 @@
+const Event = require('../models/Event')
+module.exports = {
+  createEvent: async (req, res, next) => {
+    const event = new Event(req.body)
+    try {
+      await event.save()
+      res.status(201).json({ event: event })
+    } catch (error) {
+      res.status(400).json({ error: error.errors })
+    }
+  },
+  updateEvent: async (req, res) => {
+    Event.findById(req.params.id, (err) => {
+      if (err) {
+        res.status(400).json({ error: 'Did not find an event!' })
+      } else {
+        Event.findByIdAndUpdate(req.params.id, req.body, function (err, updatedEvent) {
+          if (err) {
+            res.status(400).json({ error: 'Cannot Update the data' })
```

Comment:
you should use status in utils/status-code.js

Reply:
si, should i use npm dependecy **http-status-codes** for status handling?
other prs already using it so suppose i should use it here too

Reply:
yes you can use that